setup/functions.sh

```diff
@@ -223,3 +221,8 @@ function git_clone {
 	mv $TMPPATH/"$SUBDIR" "$TARGETPATH"
 	rm -rf $TMPPATH
 }
+
+function php_version {
+	php --version | head -n 1 | cut -d " " -f 2 | cut -c 1-3
```

Comment:
Similarly `/usr/bin/php -r "echo PHP_MAJOR_VERSION . '.' . PHP_MINOR_VERSION;"`

Reply:
I removed this bit, as I'm reverting to hardcoding the version